src/battle-scene.ts

```diff
@@ -792,10 +792,11 @@ export default class BattleScene extends SceneBase {
 
   /**
    * Returns the ModifierBar of this scene, which is declared private and therefore not accessible elsewhere
+   * @params isEnemy {@linkcode Boolean} accesses BattleScene.enemyModifierBar if true - optional, defaults to returning BattleScene.modifierBar if not present
```

Comment:
you dont need to add type for primitive values
```suggestion
   * @param isEnemy Whether to return the enemy's modifier bar
```